Content.Client/Kudzu/KudzuVisualizerSystem.cs

```diff
@@ -0,0 +1,24 @@
+using Content.Shared.Kudzu;
+using Robust.Client.GameObjects;
+
+namespace Content.Client.Kudzu
+{
+
+    public sealed class KudzuVisualsSystem : VisualizerSystem<KudzuVisualizerComponent>
+    {
+        protected override void OnAppearanceChange(EntityUid uid, KudzuVisualizerComponent component, ref AppearanceChangeEvent args)
+        {
+
+            if (!TryComp(uid, out SpriteComponent? sprite))
+            {
+                return;
+            }
+            if (args.Component.TryGetData(KudzuVisuals.Variant, out int var)
+                && args.Component.TryGetData(KudzuVisuals.GrowthLevel, out int level))
+            {
+                sprite.LayerMapReserveBlank(component.Layer);
+                sprite.LayerSetState(0, $"kudzu_{level}{var}");
```

Comment:
```suggestion
                args.Sprite.LayerMapReserveBlank(component.Layer);
                args.Sprite.LayerSetState(0, $"kudzu_{level}{var}");
```

Reply:
with the changes above

Reply:
There's actually also a problem here in the original visualizer. It should really be:
```cs
var index = args.Sprite.LayerMapReserveBlank(component.Layer);
args.Sprite.LayerSetState(index, $"kudzu_{level}{var}");
```